### ISSUE TITLE
Fix language and tests in Travis

### ISSUE THREAD

**Issue by dfdx** [JuliaGPU/CLFFT.jl#6]
1. Travis now supports `language: julia` entry, so it's possible to replace previous `language: cpp` and manual julia installation with this entry. 
2. Currently tests are failing on Travis, would be nice to fix. 

I made a couple of attempts to fix it in `new-travis-infrastructure` branch, but since I don't use CLFFT package itself, I leave it to someone with appropriate skills and configured environment.

**Comment by dfdx** [comment]
Some updates on this. I've fixed one issue related to building clFFT itself in `travis-fix-branch`, but a couple of tests - namely "Plan" and "Example FFT Single" - take long without producing any output, so Travis kills it (e.g. see [this](https://travis-ci.org/JuliaGPU/CLFFT.jl/builds/150182039) build). 

@juliohm Could you please take a look at that log and tell if you see any cues about the issue? Would be really nice to make sure there no bugs before tagging a new version.

**Comment by juliohm** [comment]
Hi @dfdx you mean pull the latest master and try to perform the tests on my local machine? In the logs I only see Travis complaining about lack of I/O as you mentioned. I will try to pull the latest and see if these tests pass on my laptop. They were not passing before, but that is because of the driver I am using probably (Beignet for integrated Intel GPU on my Dell XPS 13)

**Comment by juliohm** [comment]
Yes, unfortunately for me the test fails in the very beginning, it is a driver issue.

I think we should tag a new version regardless of the build in Travis, it was not working before anyways. The new tag will at least solve the incorrect list of supported radices.

**Comment by dfdx** [comment]
Agree. I'll clean up my experiments and merge partial fix for tests tomorrow and tag a new version just after that.

**Comment by dfdx** [comment]
@juliohm FYI: PR for a new tag is created (see JuliaLang/METADATA.jl#5853), once it's merged, new version will be available for installation.

**Comment by juliohm** [comment]
Thank you Andrei, I will specify the new tag in my package.

Best,
-Júlio
